components/qma6100p/test_apps/main/idf_component.yml

```
@@ -1,6 +1,6 @@
 ## IDF Component Manager Manifest File
 dependencies:
-  idf: ">=4.0"
+  idf: ">=5.4"
```

```suggestion
  idf: ">=5.2"
```
Driver-NG was added in idf v5.2